### DISCUSSION TITLE
pinned projects need obvious icon change when active

### DISCUSSION THREAD

**Discussion by ndftz** [uxbox/uxbox#682] [Ideas]
Took me a while to notice I had pinned my first project.  This should make it more obvious

```
.pin-icon.active{
    transform:rotate(-45deg); 
    transition:all .15s ease
}
```

While we are at it, the first time you pin something it might make sense to draw minor attention to the projects sidebar by sliding it up. 
The CSS below only affects the first pinned project. 

```
.sidebar-nav > :first-child:last-child{animation:show-up .3s ease forwards}
@keyframes show-up{0%{opacity:0; transform:translate(0,1em)}}
```

just 2cents

**Comment by myfunnyandy** [comment]
Hi, we already had an issue about this that I just promoted to [user story](https://tree.taiga.io/project/penpot/us/1298?no-milestone=1) as you are not the first one that brings this up :)

And thanks for the suggestion for the solution, I've attached this discussion to the US so we can have it in consideration once we tackle this.

**Reply by hirunatan** [reply]
This is done, we have already made the pin icon more visible.
